test/e2e/common.go

```diff
@@ -56,9 +56,12 @@ func dumpSpecResourcesAndCleanup(ctx context.Context, specName string, clusterPr
 
 	if !skipCleanup {
 		Byf("Deleting cluster %s/%s", cluster.Namespace, cluster.Name)
-		framework.DeleteClusterAndWait(ctx, framework.DeleteClusterAndWaitInput{
```

Comment:
We can delete DeleteClusterAndWait()?

Reply:
No, due to compatibility rules, but also because in the future we would like to get back to use it